### ISSUE TITLE
No movie/tv show rating if library scan only local nfo (no metadownloader)

### ISSUE THREAD

**Issue by mquan86** [jellyfin/jellyfin#4887]
There is no movie/tv show rating if the library scan using only local nfo file (Kodi format)

**System (please complete the following information):**
 - OS: Windows 10
 - Jellyfin Version: 10.6.2

**To Reproduce**

1. A movie/tv show with nfo file contains information as Kodi format (https://kodi.wiki/view/NFO_files/Movies).
2. There is rating field in nfo file.
3. In library setting, remove all Movie metadata downloaders (means use local info only).
4. Scan library.

**Expected behavior**
New added movie/tv show should have rating info

**Screenshots**
![2020-12-30_17-47-55](https://user-images.githubusercontent.com/13567269/103334897-41ee0f80-4ac7-11eb-86ee-935f39320a66.png)

**Additional context**
With my limited knowledge, I guess the problem seems the parser assume it is a simple xml tag value while in fact it is multiple complex sub tags.

From MediaBrowser.XbmcMetadata\Parsers\BaseNfoParser.cs (assume Jellyfin use this to scan local nfo file - I'm unsure)
```csharp
                case "rating":
                    {
                        var rating = reader.ReadElementContentAsString();

                        if (!string.IsNullOrWhiteSpace(rating))
                        {
                            // All external meta is saving this as '.' for decimal I believe...but just to be sure
                            if (float.TryParse(rating.Replace(',', '.'), NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture, out var val))
                            {
                                item.CommunityRating = val;
                            }
                        }

                        break;
                    }
```

Real life example of rating tags:
```xml
  <ratings>
    <rating default="true" max="10" name="imdb">
      <value>7.7</value>
      <votes>260935</votes>
    </rating>
  </ratings>
```

**Comment by lbnt** [comment]
Hi,
I can confirm the bug.
I used exclusively nfo files for metadata and Jellyfin grabs no rating.
I have changed one nfo to the use the simple <rating> tag and Jellyfin grab the rating.

In fact it's not a bug but just that Jellyfin does not implement the last nfo format specification (if we consider that the nfo format description in the Kodi wiki is a specification!).
The more complex <ratings> tag has been introduced in the kodi wiki in 2017.

The nfo parser in Jellyfin seems to be outdated and has several confirmed bugs : #4250, #3646, ...

Mat
